### PR TITLE
[APRIL FOOLS] Turns the Quartermaster into a head of staff

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

the quartermaster is now a head of staff, he gets a telescopic baton, additional access and head pda/radio
hop loses some supply access
i tested it works

## Why It's Good For The Game

clearly our man quartermaster needs some recognition, he fought valiantly for many years and still isnt a head of staff

## Changelog
:cl:
add: The Quartermaster is now a head of staff!
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
dont bully me for amount of commits i made most of this pr in july when i was still babby coder

**Comment by TiviPlus** [comment]
Wtf maintainers???????? close PR?????????
Hello???????????????

(haha 14 commits)

**Comment by trollbreeder** [comment]
have you tested this with nonhumans signing up as QM? please tell me they're allowed to be QM please tell me they're allowed to be QM

**Comment by Fikou** [comment]
man... FUCK appveyor

**Comment by Dennok** [comment]
Name it "Head of Stuff"

**Comment by HomocidlePancakes** [comment]
Wasn’t this already done for april fools?

**Comment by cacogen** [comment]
the joke is this is a permanent change

**Comment by MrDoomBringer** [comment]
fantastic PR

File: code/modules/jobs/job_types/quartermaster.dm [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
-	department_head = list("Head of Personnel")
+	auto_deadmin_role_flags = DEADMIN_POSITION_HEAD
+	department_head = list("Captain")
 	department_flag = CIVILIAN
+	head_announce = list(RADIO_CHANNEL_SUPPLY)
 	faction = "Station"
 	total_positions = 1
 	spawn_positions = 1
-	supervisors = "the head of personnel"
+	supervisors = "absolutely no one"
```
Comment (ATH1909) [inline code/modules/jobs/job_types/quartermaster.dm]: NO GODS, NO MASTERS

File: code/modules/jobs/job_types/quartermaster.dm [hunk 2]
```diff
@@ -1,20 +1,26 @@
 /datum/job/qm
 	title = "Quartermaster"
 	flag = QUARTERMASTER
-	department_head = list("Head of Personnel")
+	auto_deadmin_role_flags = DEADMIN_POSITION_HEAD
+	department_head = list("Captain")
```
Comment (ATH1909) [inline code/modules/jobs/job_types/quartermaster.dm]: below, it says that "absolutely no one" is the QM's supervisor, but this seems to contradict that

File: code/modules/jobs/job_types/quartermaster.dm (hunk 1 above)
Comment (trollbreeder) [reply code/modules/jobs/job_types/quartermaster.dm]:
```suggestion
	supervisors = "technically the head of personnel but who the hell cares go do whatever"
```

File: code/modules/jobs/job_types/quartermaster.dm (hunk 2 above)
Comment (trollbreeder) [reply code/modules/jobs/job_types/quartermaster.dm]:
```suggestion
	department_head = null // NO GODS, NO MASTERS
```

Comment (Fikou) [reply code/modules/jobs/job_types/quartermaster.dm]: i think i need to put a job there

**Comment by Fikou** [comment]
> Don't forget to make the QM count as a head of staff for the purposes of revs and not being allowed to be nonhuman.

already included in being a command position i think?

**Comment by IndieanaJones** [comment]
> Name it "Head of Stuff"

I think you're referring to the Halfmaster.

**Comment by nikothedude** [comment]
this had so much potential, if you had made the qm just a head, like litterally jsut a fucking head, it would probably have been merged

**Comment by wesoda25** [comment]
@nikothedude doesnt need to be open to be test merged

**Pull request closed without merging**